src/NLU.DevOps.Luis/LuisClient.cs

```diff
@@ -5,28 +5,18 @@ namespace NLU.DevOps.Luis
 {
     using System;
     using System.Collections.Generic;
-    using System.Net;
-    using System.Net.Http;
-    using System.Text;
     using System.Threading;
     using System.Threading.Tasks;
-    using Logging;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring;
     using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Models;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime;
     using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime.Models;
-    using Microsoft.CognitiveServices.Speech;
-    using Microsoft.CognitiveServices.Speech.Audio;
-    using Microsoft.CognitiveServices.Speech.Intent;
-    using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
 
     internal class LuisClient : ILuisClient
     {
-        private const string Protocol = "https://";
-        private const string Domain = ".api.cognitive.microsoft.com";
+        public const string Protocol = "https://";
+        public const string Domain = ".api.cognitive.microsoft.com";
 
-        private static readonly TimeSpan ThrottleQueryDelay = TimeSpan.FromMilliseconds(100);
+        private readonly ILuisTrainingClient trainingClient;
+        private readonly ILuisPredictionClient predictionClient;
 
         public LuisClient(
```

Comment:
Is the eventual goal going to be to have a .ctor for `LuisClient` that takes in the two `ILuis*Client` interfaces?

Reply:
Or - is the `LuisClient` only staying around temporarily to minimize the diff? Seems like it's becoming redundant.

Reply:
Yes the goal would be that we could dependency inject the different interfaces into the LuisClient constructor, but yes i could remove it later as it is redundant. That would be a bigger refactor since its used everywhere and i wanted to minimize diff as you said